junit-platform-console/src/main/java/org/junit/platform/console/options/AvailableOptions.java

```diff
@@ -150,6 +152,16 @@
 			"Select a classpath resource for test discovery. This option can be repeated.") //
 				.withRequiredArg();
 
+		// --- Java Platform Module System -------------------------------------
+
+		scanModulePath = parser.acceptsAll(asList("scan-modulepath", "scan-module-path"), //
+			"EXPERIMENTAL: Scan all modules on the module-path for test discovery.");
+
+		selectedModules = parser.acceptsAll(asList("o", "select-module"), //
+			"EXPERIMENTAL: Select single module for test discovery. This option can be repeated.") //
+				.withRequiredArg() //
+				.describedAs("module name");
+
```

Comment:
Should we also add support in the Vintage engine?

Reply:
Mh? Vintage should support both new selectors. If it doesn't then something was lost in rebasing to master.

Reply:
Find vintage-affecting changes below, around https://github.com/junit-team/junit5/pull/1061/files#diff-5e437a25c16596592cf66086916f6fed

Reply:
Sorry, missed that! LGTM!